app/views/common/_menu.html.haml

```diff
@@ -1,14 +1,22 @@
-.row
-  .col-md-10
-    %ul.nav.nav-pills
-      %li
-        = link_to 'Главная', root_path
-      %li
-        = link_to 'Список хранилищ', storages_path
-      %li
-        = link_to 'Новое хранилище', new_storage_path
-  .col-md-2
-    .pull-left
-      = "Вы вошли как: <b>#{current_user.full_name}</B>".html_safe
-    .pull-right
-      = link_to 'Выйти', destroy_user_session_path, method: :delete
\ No newline at end of file
+%nav.navbar.navbar-default.navbar-static-top
+  .container
+    .row
+      .col-md-9
+        %ul.nav.navbar-nav
+          %li
+            = link_to 'Главная', root_path
+          %li
+            = link_to 'Список хранилищ', storages_path
+          %li
+            = link_to 'Новое хранилище', new_storage_path
+      .col-md-3
+        .dropdown
```

Comment:
Line is too long. [177/120]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace